evals/registry/evals/slope_intercept.yaml

```diff
@@ -0,0 +1,9 @@
+slope_intercept:
+  id: slope_intercept.dev.v0
+  description: Test a model on creating slope-intercept equations from given points.
+  metrics: [accuracy]
+
+slope_intercept.dev.v0:
+  class: evals.elsuite.basic.match:Match
```

Comment:
You may want to try includes here unless you think the ability to output the answer concisely without showing any work is critical to the evaluation

Reply:
I think it might be - to make the model act more like a calculator instead of explaining its work in some situations might be very necessary (for example, an assistant in an industrial calculator app should not output work unless asked for it). I have compromised - I am doing `Includes`, but asking the model not to output its work. I would like to see how it performs solely outputting the answer without including calculations.

Reply:
Update: I have changed this to `Match`, as `Includes` was producing incorrect results.

Reply:
I think if the model is explaining the results then the prompt needs to be more explicit that no explanations are allowed

Reply:
I agree. The prompt now contains this:
> do not explain the equation or provide any other information.

Should I restate this in the prompt to make it more clear?

Reply:
I'm not a prompt engineer so I wouldn't know :P BTW you still have a .DS_Store in the root